components/org.wso2.carbon.identity.handler.event.account.lock/pom.xml

```diff
@@ -61,9 +61,24 @@
                         </Export-Package>
                         <Import-Package>
                             org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
-                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}"
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.apache.commons.lang.*; version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging; version="${commons-logging.osgi.version.range}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
+                            org.wso2.carbon.user.core.*; version="${carbon.kernel.version}",
+                            org.wso2.carbon.identity.application.common.*; version="${project.version}",
```

Comment:
Should be,
`org.wso2.carbon.identity.application.common.*;
                            version="${carbon.identity.framework.imp.pkg.version.range}",`